### ISSUE TITLE
The concept of Equalized Learning Rate

### ISSUE THREAD

**Issue by ken881015** [rosinality/stylegan2-pytorch#359]
I draw a flow chart of EqLR according to the implementation method of code, can someone discuss with me the correctness of my thought?

![image](https://github.com/user-attachments/assets/ffd0f1be-6c40-4ab8-a323-bb1289751830)
